Update configuration.md (#1064)
viamrobotics/docs: docs/manage/configuration.md

{{< alert title="Tip" color="tip" >}}
You can also store your config file in another location.

> To change where `viam-server` looks for its config file, run the following command:

Author: ```suggestion
To start `viam-server` with a config file in a different location, run the following command:
```

@JessamyT is this better?
